src/lib/util.ts

```diff
@@ -64,7 +64,7 @@ export function toTitleCase(str: string) {
 	return splitStr.join(' ');
 }
 
-export function cleanString(str: string) {
+export function cleanString(str = '') {
```

Comment:
why?

Reply:
so that if undefined is passed it doesnt just halt execution, which is what happens when you call a command with no parameters